roles/tools/tasks/write_pmm_kubectl_config.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Checking for .kube config file
+  ansible.builtin.stat:
+    path: "{{ tools_kubectl_config_file }}"
+  register: tools_kubectl_config_check
+
+- name: Set tools_kubectl_env_config_file default
+  ansible.builtin.set_fact:
+    tools_kubectl_env_config_file: "{{ tools_kubectl_config_file }}"
+
+- name: Set tools_kube_config_name
+  ansible.builtin.set_fact:
+    tools_kubectl_env_config_file: "{{ tools_kubectl_config_file }}_{{ item }}"
+  when: tools_kubectl_config_check.stat.exists
+
+- name: Retrieve kube config from pmm
+  ansible.builtin.uri:
+    url: "https://{{ pmm_server_host }}:{{ pmm_server_port }}/v1/management/DBaaS/Kubernetes/Get"
```

Comment:
@aaronplagge please use the predefined path fragments, in this case it would be either `pmm_api_management` or a new one that extends to include `Kubernetes`

Reply:
@aaronplagge any updates?